### ISSUE TITLE
Failed to compile with nightly

### ISSUE THREAD

**Issue by ihrwein** [reem/stainless#31]
Hi!

First of all, thanks for this great library :)

I tried to use it with the current nighly but got an error:

```
/home/tibi/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/stainless-0.0.18/src/parse.rs:104:30: 104:38 error: no method named `as_str` found for type `syntax::ast::Ident` in the current scope
/home/tibi/.multirust/toolchains/nightly/cargo/registry/src/github.com-0a35038f75765ae4/stainless-0.0.18/src/parse.rs:104             match block_name.as_str() {
```

Rustc version:
```
$ rustc --version
rustc 1.5.0-nightly (f5a015892 2015-10-01)
```

**Comment by ihrwein** [comment]
Your current master is OK so perhaps we miss a new release :)

**Comment by reem** [comment]
Released 0.1.0!

**Comment by ihrwein** [comment]
Thanks!
